src/collection-view.js

```diff
@@ -207,11 +207,13 @@ const CollectionView = Backbone.View.extend({
   // Render children views. Override this method to provide your own implementation of a
   // render function for the collection view.
   render() {
-    if (this._isDestroyed) { return this; }
-    this.triggerMethod('before:render', this);
-    this._renderChildren();
-    this._isRendered = true;
-    this.triggerMethod('render', this);
+    if (!this._isDestroyed) {
```

Comment:
yeah the rest looks fine to me.

As far as coverage is concerned we have line and function coverage at 100% now.. this issue is a statement and branch issue, but this change in the code doesn't resolve the branch coverage.. for full coverage we'd need to test the if and else of the condition.. changing the `if` wrapping doesn't really fix coverage.